en/flight_modes/offboard.md

```diff
@@ -56,14 +56,28 @@ The action is defined in the parameters [COM_OBL_ACT](#COM_OBL_ACT) and [COM_OBL
 ### Fixed Wing
 
 * [SET_POSITION_TARGET_LOCAL_NED](https://mavlink.io/en/messages/common.html#SET_POSITION_TARGET_LOCAL_NED)
-  * The following input combinations are supported: <!-- https://github.com/PX4/Firmware/blob/master/src/lib/FlightTasks/tasks/Offboard/FlightTaskOffboard.cpp#L166-L170 -->
+  * The following input combinations are supported (via `type_mask`): <!-- https://github.com/PX4/Firmware/blob/master/src/lib/FlightTasks/tasks/Offboard/FlightTaskOffboard.cpp#L166-L170 -->
     * Position setpoint (only `x`, `y`, `z`)
-  - PX4 supports the coordinate frames (`coordinate_frame` field): [MAV_FRAME_LOCAL_NED](https://mavlink.io/en/messages/common.html#MAV_FRAME_LOCAL_NED) and [MAV_FRAME_BODY_NED](https://mavlink.io/en/messages/common.html#MAV_FRAME_BODY_NED).
-
+  * Specify the *type* of the setpoint in `type_mask` (not part of the MAVLink standard).
+    The values are:
+    - Following bits not set: vehicle will fly in a flower-like pattern.

```

Comment:
For fixed wing, the type masks are set from https://mavlink.io/en/messages/common.html#POSITION_TARGET_TYPEMASK. Unless `POSITION_TARGET_TYPEMASK_X_IGNORE`, `POSITION_TARGET_TYPEMASK_Y_IGNORE`, `POSITION_TARGET_TYPEMASK_Z_IGNORE` are set. This will make the position setpoints to be ignored.

Reply:
@Jaeyoung-Lim I am not certain what point you are making here. Is it it:
- That the pattern flown is not a flower-like pattern - ie it follows the input setpoings
- That all of these types are ignored if you're not a position setpoint
- Something else?

Just trying to understand what is wrong with what is written.

Reply:
@Jaeyoung-Lim Can you clarify. I'd like to push this in to docs early this week.